lib/tizen_sdk.dart

```diff
@@ -250,22 +250,36 @@ class TizenSdk {
     String? apiVersion,
     required String arch,
   }) {
-    if (profile == 'common') {
-      // Note: The headless profile is not supported.
-      profile = 'iot-headed';
-    }
-    if (profile == 'tv') {
-      // Note: The tv-samsung rootstrap is not publicly available.
-      profile = 'tv-samsung';
-    }
```

Comment:
I think this conversion is quite reasonable. Is there any rootstrap with the profile name "tv" in the real world?